service/rtc/session.go

```diff
@@ -145,11 +145,11 @@ func (s *session) getSourceRate(rid string) int {
 	return rate
 }
 
-func (s *session) getOutScreenTrack(rid string) *webrtc.TrackLocalStaticRTP {
+func (s *session) getOutScreenTrack(mimeType, rid string) *webrtc.TrackLocalStaticRTP {
 	s.mut.RLock()
 	defer s.mut.RUnlock()
 
-	return pickRandom(s.outScreenTracks[rid])
+	return pickRandom(s.outScreenTracks[getTrackIndex(mimeType, rid)])
```

Comment:
Just curious, why is it pickRandom?

Reply:
@cpoile Because of https://github.com/mattermost/rtcd/pull/138 if you recall